### DISCUSSION TITLE
edsr训练不收敛

### DISCUSSION THREAD

**Discussion by huoshuai-dot** [open-mmlab/mmediting#1188] [Models] [answered]
数据集采用tools/data/super-resolution/preprocess_div2k_dataset.py生成训练数据集（X4 mild），config文件使用的全部都是默认参数，但是 训练的时候不收敛 生成图片效果很差 不明白 这是什么原因 按照常理来讲 L1 loss应该是比较好训练的 不知道 有没有大佬遇见过同样的问题呢？

**Comment by Yshuo-Li** [comment, marked as answer]
最近使用mmediting训练过EDSR，下采样x4，是收敛的，不过使用的数据集不是来自tools/data/super-resolution/preprocess_div2k_dataset.py，建议楼主检查一下生成的数据集效果。

另外EDSR的四倍下采样是在二倍下采样预训练基础上做的。

**Comment by huoshuai-dot** [comment]
> 最近使用mmediting训练过EDSR，下采样x4，是收敛的，不过使用的数据集不是来自tools/data/super-resolution/preprocess_div2k_dataset.py，建议楼主检查一下生成的数据集效果。
> 
> 另外EDSR的四倍下采样是在二倍下采样预训练基础上做的。

问题明白了，官方给的默认参数基于的训练数据是bicubic降采样出来的数据，我之前搞的是DIV2K 2018放出来的那些数据 所以跟官方log匹配不上 现在换了数据就匹配上了，3q
